easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.0-foss-2017b-Python-3.6.2.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.0'
+versionsuffix = '-Python-%(pyver)s'
+
+libpng_ver = '1.6.32'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '3.6.2'),
+    ('libpng', libpng_ver),
```

Comment:
If you use freetype 2.8 (which depends on `libpng` `1.6.32`, you can drop `libpng_ver` and just use `'1.6.32'` here

Reply:
I did not realise that it was an equivalent libpng enabled version.  I will change that.

Reply:
The reason that some `freetype` easyconfigs have `-libpng-<version>` in the `versionsuffix` is because there was a need to update `libpng` somewhere else, where `freetype` is also required, and the existing `freetype` easyconfigs at the time used an older `libpng` as dependency...